html-tables/index.html

```diff
@@ -6,11 +6,11 @@
   <body>
     <h1>Student Grade Table</h1>
     <table>
-      <tr>
+      <thead>
```

Comment:
You still need a `tr`, it just needs to be a child of the `thead` element. You should review the documentation for this exercise again, you seem to have an incomplete understanding of the concept.

Reply:
Went back to review and now understand fully, thank you.